integration_tests/features/Propagation.feature

```diff
@@ -50,3 +50,35 @@ Feature: Block Propagation
     Then I receive an error containing 'Orphan block'
     And all nodes are at height 1
 
+  Scenario: Nodes should never sync but keep insync
```

Comment:
Huh?

Reply:
"Nodes should not switch to block sync state, but should instead stay synced through block propagation"